lightbluetent/home.py

```diff
@@ -55,10 +63,16 @@ def home():
     if user.societies:
         user_societies = user.societies
 
-        return render_template("home/home.html", page_title="Home", user_societies=user_societies, crsid=crsid)
+        running_meetings = {}
+
+        for society in user.societies:
+            meeting = AttendeeMeeting(society.bbb_id, society.attendee_pw, "")
+            running_meetings[society.bbb_id] = meeting.is_running()
+
+        return render_template("home/home.html", page_title="Home", user_societies=user_societies, running_meetings=running_meetings, crsid=crsid)
 
     else:
-        return render_template("home/home.html", page_title="Home", user_societies={}, crsid=crsid)
+        return render_template("home/home.html", page_title="Home", user_societies={}, running_meetings={}, crsid=crsid)
```

Comment:
We should validate the params on the jinja side instead of passing in an empty object. Either don't pass the param when not needed or make them an empty object before so you only have to make one `render_template` call here.